### DISCUSSION TITLE
ADB截图时的文件交换，是在哪里进行的呀？

### DISCUSSION THREAD

**Discussion by HS7097** [LmeSzinc/AzurLaneAutoScript#1744] [Q&A]
我在用Alas测试新队伍能否稳定13/4的时候，发现模拟器会在截图时卡一下；关闭ADB链接，手操就不会有问题。
按照我的电脑配置，这种问题不应该发生的，这虽然不影响使用，但我想看有没有办法排除这个问题

**Comment by LmeSzinc** [comment]
https://github.com/LmeSzinc/AzurLaneAutoScript/blob/28de1c51e238993ac0f42c7bc88e1197207a61b0/module/device/method/adb.py#L127-L151

**Comment by HS7097** [comment]
妈耶，测试了好多遍，应该是错误报告与错误截图的问题

**Comment by LmeSzinc** [comment]
你需要完整描述你做了什么，才会有人能够帮到你。

**Reply by HS7097** [reply]
> 你需要完整描述你做了什么，才会有人能够帮到你。

首先发现了Alas接管后会导致模拟器与截图同频率卡顿，类似于抽帧那种，有闪现甚至会导致游戏炮弹黏在一起，并且发生了Alas罢工的问题
毕竟也不懂代码嘛，翻完之后没发现路径信息，因此推测与是否在同一磁盘关联不大
但为了保险起见在不同电脑上试了好多次，测试模拟器包括蓝叠全版本，夜神，雷电9，期间Alas均在模拟器默认安装路径下
测试工具为微星小飞机的OSD，主要数据为帧时间，FPS，low0.1%
只有雷电9在任何情况下手操与Alas接管近乎没有差别，也是受这个问题影响最小的
蓝叠所有版本与夜神在长时间由Alas接管后都会导致模拟器卡顿，其中最严重的，也是我最常用的版本蓝叠4 32 ，能卡到拖不动地图。在调小保存错误前截图这个数值之后会有轻微改善，但仍不能从根源解决卡顿问题
测试机器有 12900K 32G 3090ti Windows 10 21H2，9700kf 16g RX590 Windows Server 2019

**Comment by HS7097** [comment]
雷电稳定性也算不错了，连续运行5天了，还可以，上下翻转的问题好像雷电9不存在
